tools/xctoolrunner/xctoolrunner.py

```diff
@@ -213,6 +213,16 @@ def actool(_, toolargs):
 
   xcrunargs += toolargs
 
+  # The argument coming after "--compile" is the output directory. "actool"
+  # expects an directory to exist at that path. Create an empty directory there
+  # if one doesn't exist yet.
+  for idx, arg in enumerate(toolargs):
+    if arg == "--compile":
+      output_dir = toolargs[idx + 1]
+      if not os.path.exists(output_dir):
```

Comment:
could use `os.makedirs(output_dir, exist_ok=True)` instead of checking for existence

Reply:
Checked that but it's only available on Python ≥ 3.5.

Reply:
ugh i wish rules_python just bundled real python toolchains...